Radiant/fileWatcher.js

```diff
@@ -152,16 +152,18 @@ const uploadFile = function (info, endStream){
                 } else {
                     const pathFind = info.path.match(/^(.*[\\\/])/);
                     const mainPath = pathFind[0].substr(0, pathFind[0].length - 1);
-
+                    const thumbnailKey = data.Key.split('-')[0];
+                    const segment = data.Key.split('-')[1];
                     if(ext === 'm3u8' && _.has(streamTracker[info.path], 'm3u8') && !streamTracker[info.path].m3u8){
                         streamTracker[info.path].m3u8 = true;
                         setTimeout(() => {
                             Logger.log(`CREATING VIDEO STREAM - conversationTopicId = ${streamTracker[info.path].conversationTopicId} fileKey = ${info.path.replace(/^.*[\\\/]/, '')} `);
-                            const thumbnailKey = data.Key.split('-')[0];
+                            // const thumbnailKey = data.Key.split('-')[0];
```

Comment:
can this be deleted?

Reply:
no... it NEEDS to be there.. jk ill remove it